zeppelin-integration/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java

```diff
@@ -674,7 +674,7 @@ public void testMultipleDynamicFormsSameType() throws Exception {
     }
   }
 
-  @Test
+//  @Test
```

Comment:
are these tests broken? where are they disabled?

Reply:
oops, I mix this PR with another PR for ZEPPELIN-3535. Will revert these changes